src/sentry/plugins/bases/notify.py

```diff
@@ -96,6 +92,11 @@ def rule_notify(self, event, futures):
                 rules=rules,
             )
             self.notify(notification)
+            self.logger.info('notification.dispatched', extra={
```

Comment:
Shouldn't this come before `notify()` ? If `notify()` were to raise, this would never be logged.